app/models/group.rb

```diff
@@ -142,7 +142,7 @@ def self.visibility_levels
   scope :with_smtp_configured, -> { where(smtp_enabled: true) }
 
   scope :visible_groups, Proc.new { |user, order, opts|
-    groups = self.order(order || "name ASC")
+    groups = self.order(order || "groups.name ASC")
```

Comment:
This allows the query to be merged into other queries which may contain a table with a 'name' column